### ISSUE TITLE
Try EfficientNet

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#58]
Looks like it's in RC2. Try when the release happens and see how it performs. Ideally we'd try MobileNetV3 but it's not out yet.

**Comment by wingman-jr-addon** [comment]
Gonna try this out with tf-nightly.

**Comment by wingman-jr-addon** [comment]
When I use tf-nightly, tfa-nightly is not happy. TFA importantly provides image rotation. I will just wait for TF 2.2.0. It's now on RC3.

**Comment by wingman-jr-addon** [comment]
In theory, this should have been [in TF 2.2.0](https://github.com/tensorflow/tensorflow/blob/v2.2.0/tensorflow/python/keras/applications/efficientnet.py), but API docs show it's only in nightly and it does not import when I try it.
I decided to take another run at this with [qubvel's implementation](https://github.com/qubvel/efficientnet).
EfficientNet didn't yield any benefits on B0.
Trying B1 now. So far it looks like either no benefit or only modest gains will be achieved.

**Comment by wingman-jr-addon** [comment]
Tried B1. Still not really seeing much gain.
Trying B4. This will take quite some time to train.

**Comment by wingman-jr-addon** [comment]
Putting B4 on hold. I can train it but its rate and progress without any clear hint there'd be a payoff that I want to try other experiments first.

**Comment by wingman-jr-addon** [comment]
I think with advancements into e.g. MobileNetV3 etc. I'm going to close for now.

**Comment by wingman-jr-addon** [comment]
So I tried this again using B4 as a finetune and after greatly increasing regularization and backing off the learning rate, I started to see better results. I am also trying B0 again with the same hyperparameters to see if that makes a difference and am observing a possible improvement there as well.

**Comment by wingman-jr-addon** [comment]
So ultimately I found and tried a variant on this with a finetune of EfficientNet-Lite0, which became SQRXR 107 as tracked on the general #64 thread via PR #91 .
